File: 백준/Bronze/10810. 공 넣기/공 넣기.go
package main

import (
	"fmt"
	"strconv"
	"strings"
)

func main() {
	var n, m int

	fmt.Scanln(&n, &m)

	b := make([]string, n)

	for i := range b {
		b[i] = "0"
	}

	var i, j, k int

	for x := 0; x < m; x++ {
		fmt.Scanln(&i, &j, &k)

		for z := i; z < j+1; z++ {
			b[z-1] = strconv.Itoa(k)
		}
	}

	output := strings.Join(b, " ")
	fmt.Println(output)
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
n, m = map(int,input().split())
b = [0] * n

for _ in range(m):
    i,j,k = map(int,input().split())
    for x in range(i, j+1):
        b[x-1] = k

for x in range(n):
    print(b[x], end=" ")